src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import StyledAvatar from './styled/avatar';
+import StyledImage from './styled/image';
+import StyledText from './styled/text';
+import StyledIcon from './styled/icon';
+import RenderIf from '../RenderIf';
+import { BaseProps } from '../types';
+
+type AvatarSizes = 'large' | 'medium' | 'small' | 'x-small';
+
+interface Props extends BaseProps {
+    icon?: ReactNode;
+    initial?: string;
+    src?: string;
+    size?: AvatarSizes;
+    style?: object;
```

Comment:
You don't need to specify style prop since it is already defined in BaseProps and this interface extends from BaseProps.

Reply:
done